pyproject.toml

```diff
@@ -43,6 +43,7 @@ dependencies = [
     "packaging>=20.0",
     "pandas>=1.1.5,<2.3",
     "psutil",
+    "py-cpuinfo>=9.0.0",
```

Comment:
@KodiaqQ, @MaximProshin, please pay attention to this in view of release activities.

Reply:
SDL passed. This dep is used in OV so I don't see any issue with it.